h2o-py/h2o/estimators/estimator_base.py

```diff
@@ -107,14 +108,101 @@ def train(self, x=None, y=None, training_frame=None, offset_column=None, fold_co
         :param float max_runtime_secs: Maximum allowed runtime in seconds for model training. Use 0 to disable.
         :param bool verbose: Print scoring history to stdout. Defaults to False.
         """
-        self._train(x=x, y=y, training_frame=training_frame, offset_column=offset_column, fold_column=fold_column,
-                    weights_column=weights_column, validation_frame=validation_frame, max_runtime_secs=max_runtime_secs, 
-                    ignored_columns=ignored_columns, model_id=model_id, verbose=verbose)
+        parms = self._make_parms(x=x, y=y, training_frame=training_frame, offset_column=offset_column, 
+                                 fold_column=fold_column, weights_column=weights_column, 
+                                 validation_frame=validation_frame, max_runtime_secs=max_runtime_secs, 
+                                 ignored_columns=ignored_columns, model_id=model_id, verbose=verbose)
+        self._train(parms, verbose=verbose)
```

Comment:
to keep some symmetry with `bulk_train`, maybe `train` should now return `self`. wdyt?

Reply:
Makes sense, let me add that in a new pr